CHANGELOG.next.md

```diff
@@ -16,7 +16,7 @@ Thanks, you're awesome :-) -->
 
 #### Added
 
-* Add `orchestrator` fieldset to experimental schema. #1292
+* Add `orchestrator` fieldset as beta fields. #1326
```

Comment:
Editing the existing changelog entry since this RFC advanced from stage 0 to 2 within one release cycle 🚄